File: Daily Pratice Question/JAVASCRIPT/question(10).js
// 10. Given an array of integers, count how many numbers are even and how many are odd.
// Examplef
// Input: [1, 2, 3, 4, 5]
// Output: Even: 2, Odd: 3

let arr=[1, 2, 3, 4, 5]
let even=0;
let odd=0;
for(i=0;i<arr.length;i++){
    if(arr[i]%2==0){
      even++
  
    }
    odd=arr.length-even
}console.log("even number are",even)
console.log("odd number are",odd)

File: Daily Pratice Question/C LANGUAGE/Question(10).c
// 10. Given an array of integers, count how many numbers are even and how many are odd.
// Example
// Input: [1, 2, 3, 4, 5]
// Output: Even: 2, Odd: 3


#include <stdio.h>

int main() {
   int arr[] = {1, 2, 3, 4, 5};
   int even;
   int odd;
int i;
   int len=sizeof(arr)/sizeof(arr[0]);
   for(i=0;i<len;i++){
      if( arr[i]%2==0){
          even++;
          
}
     
   }printf(" even numbers are %d\n",even);
   odd=len-even;
printf("Odd number are %d",odd);
      
      return 0;
}